sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/implementation/notifications/fcm/PushNotificationClient.java

```diff
@@ -313,40 +375,25 @@ public void run() {
         this.registrationRenewScheduleTimer.schedule(task, delayMs);
     }
 
-    private void refreshEncryptionKeys() throws Throwable {
-        if (this.keyGenerator == null) {
-            this.keyGenerator = KeyGenerator.getInstance("AES");
-            this.keyGenerator.init(KEY_SIZE);
-        }
-
-        this.previousCryptoKey = this.cryptoKey;
-        this.previousAuthKey = this.authKey;
-
-        this.cryptoKey = this.keyGenerator.generateKey();
-        this.authKey = this.keyGenerator.generateKey();
-
-        this.keyRotateTimeMillis = System.currentTimeMillis();
-    }
-
     private String decryptPayload(String encryptedPayload) throws Throwable {
         this.logger.verbose(" Decrypting payload.");
-
+        Log.i("decryption", "cypto: " + renewalWorkerDataContainer.getCurAuthKey() + ", auth: " + renewalWorkerDataContainer.getCurAuthKey());
```

Comment:
We prefer to use `ClientLogger` as it already uses the Android logger by default unless otherwise specified.